src/zoid/buttons/util.js

```diff
@@ -18,6 +18,13 @@ export function determineFlow(props : ButtonProps) : $Values<typeof BUTTON_FLOW>
     }
 }
 
+export function canUseQRPay(funding : $Values<typeof FUNDING>) : boolean {
```

Comment:
Could we simplify this condition? What do you think of this?
```js
export function canUseQRPay(funding : $Values<typeof FUNDING>) : boolean {
    if (funding === FUNDING.VENMO && !isIos() && !isAndroid()) {
        return true;
    }

    return false;
}
```

Reply:
yep that seems easier to read